src/app/public/plugin-resources/code-examples/text-expand/inline/text-expand-demo.component.ts

```diff
@@ -0,0 +1,14 @@
+import {
+  Component
+} from '@angular/core';
+
+@Component({
+  selector: 'app-text-expand-demo',
```

Comment:
In the current code example, we use a selector starting with `sky` instead of `app`. Just making sure that's a deliberate change. Same question on the other 2 examples.

Reply:
Yep, that's deliberate. All the code example selectors should start with `app-`.